modules/effects/arrow-effects-reactor/src/main/kotlin/arrow/effects/FluxK.kt

```diff
@@ -80,7 +94,7 @@ data class FluxK<A>(val flux: Flux<A>) : FluxKOf<A>, FluxKKindedJ<A> {
 
     fun <A> runAsync(fa: Proc<A>): FluxK<A> =
       Flux.create { emitter: FluxSink<A> ->
-        fa { either: Either<Throwable, A> ->
+        fa(IOConnection()) { either: Either<Throwable, A> ->
```

Comment:
Can't we register the `IOConnection` to work in sync with `emitter`? AFAIK they represent the same semantics.

Reply:
`IOConnection` is not exposed through Async anymore, mimicking cats.